app/views/questionnaire.py

```diff
@@ -123,155 +129,165 @@ def save_questionnaire_store_wrapper(*args, **kwargs):
 
 @questionnaire_blueprint.route('<group_id>/<int:group_instance>/<block_id>', methods=['GET'])
 @login_required
+@with_answer_store
+@with_metadata
+@with_schema
```

Comment:
Is the order important here?
Just a thought, can we make the wrappers pass in the value as a kwarg?

Reply:
Order is very important. can easily pass as kwargs, but I was being consistent with the existing `@full_routing_path_required` decorator

Reply:
Yeh, just thinking the more we add the more brittle it could become to order, maybe if we make them all use kwarg it would be more resilient?

Reply:
would we also get funky errors if they were in the wrong order? so a schema was actually metadata?

Reply:
Yep, we would. it's weird errors either way, both ways have the same downsides I think. What do you reckon @jonnyshaw89